<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_9</class>
 <widget class="QWidget" name="Screen_9">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="d2_9_backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#d2_9_backgroundWidget{
	background-color: rgb(255, 255, 255);
	border-radius:  22px;
	
	border-color: rgb(0, 0, 0);
	border-style: outset;
	border-width: 8px;
}

#d2_9_btnClose{
	border-image: url(:/desing2/buttons/close buttons/big/close-button-normal-big.png);
}

#d2_9_btnClose::pressed{
	border-image: url(:/desing2/buttons/close buttons/big/close-button-pressed-big.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,3,6,3,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="d2_9_hlTop">
        <item>
         <spacer name="horizontalSpacer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="d2_9_btnClose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="d2_9_lbInfoText">
        <property name="font">
         <font>
          <family>Baloo Cyrillic</family>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;p style=&quot;line-height:60&quot;&gt;Извините эта игрушка 
 закончилась!&lt;p&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="d2_9_lblIllustrationNoToy">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="d2_resource/Resource.qrc">:/desing2/illustrations/no-toy-illustration.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="d2_9_lblText">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="d2_resource/Resource.qrc">:/desing2/text/no-toys-text-gradient.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="d2_9_btnTry">
            <property name="styleSheet">
             <string notr="true">#d2_9_btnTry{
	border-image: url(:/desing2/buttons/buy another toy buttons/buy-another-toy-button-normal.png);
}

#d2_9_btnTry::pressed{
	border-image: url(:/desing2/buttons/buy another toy buttons/buy-another-toy-button-pressed.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="d2_9_btnReturnVisa">
            <property name="styleSheet">
             <string notr="true">#d2_9_btnReturnVisa{
	border-image: url(:/desing2/buttons/take change buttons/take-change-button-normal.png);
}

#d2_9_btnReturnVisa::pressed{
	border-image: url(:/desing2/buttons/take change buttons/take-change-button-pressed.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="d2_9_btnBuyMore">
            <property name="styleSheet">
             <string notr="true">#d2_9_btnBuyMore{
	border-image: url(:/desing2/buttons/buy another toy buttons/buy-another-toy-button-normal.png);
}

#d2_9_btnBuyMore::pressed{
	border-image: url(:/desing2/buttons/buy another toy buttons/buy-another-toy-button-pressed.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="d2_9_lblInfoReturn">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>117</green>
                  <blue>17</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>255</red>
                  <green>117</green>
                  <blue>17</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="WindowText">
                <brush brushstyle="SolidPattern">
                 <color alpha="255">
                  <red>190</red>
                  <green>190</green>
                  <blue>190</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <family>Baloo Cyrillic</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Иформация о сдаче</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QPushButton" name="d2_9_btnGetCoin">
            <property name="styleSheet">
             <string notr="true">#d2_9_btnGetCoin{
	border-image: url(:/desing2/buttons/take change buttons/take-change-button-normal.png);
}

#d2_9_btnGetCoin::pressed{
	border-image: url(:/desing2/buttons/take change buttons/take-change-button-pressed.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="d2_resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
